README.md

```diff
@@ -14,3 +14,11 @@ Coming soon:
 - tests for our code!
 - feedback box for more questions and other suggestions
 - moderated discussion page(s) 
+
+
+## Installation instructions (development)
+
+- edit django_project/sample_settings.py into settings.py
```

Comment:
"rename `django_project/sample_settings.py` to `django_project/settings.py`" would be more clear I think